src/tests/ftest/util/test_utils_pool.py

```diff
@@ -90,6 +90,8 @@ def remove_pool(test, pool):
     if exit_status_exception is False:
         pool.dmg.exit_status_exception = exit_status_exception
 
+    sleep(120)
+
```

Comment:
In https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-12842/7 this has proven that the changes in `src/tests/ftest/util/apricot/apricot/test.py` are working correctly, e.g.
```
2023-08-24 23:52:07,304 process          L0686 INFO | Command '/usr/bin/dmg -o /etc/daos/daos_control.yml -d -j server set-logmasks' finished with 0 after 0.12249326705932617s
2023-08-24 23:53:32,105 stacktrace       L0039 ERROR| 
2023-08-24 23:53:32,105 stacktrace       L0042 ERROR| Reproduced traceback from: /usr/lib/python3.6/site-packages/avocado/core/test.py:793
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR| Traceback (most recent call last):
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|   File "/usr/lib/daos/TESTING/ftest/util/apricot/apricot/test.py", line 1393, in tearDown
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|     self._teardown_errors.extend(self._cleanup())
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|   File "/usr/lib/daos/TESTING/ftest/util/apricot/apricot/test.py", line 413, in _cleanup
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|     raise error
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|   File "/usr/lib/daos/TESTING/ftest/util/apricot/apricot/test.py", line 409, in _cleanup
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|     errors.extend(cleanup["method"](**cleanup["kwargs"]))
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|   File "/usr/lib/daos/TESTING/ftest/util/test_utils_pool.py", line 93, in remove_pool
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|     sleep(120)
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|   File "/usr/lib/python3.6/site-packages/avocado/plugins/runner.py", line 77, in sigterm_handler
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR|     raise RuntimeError("Test interrupted by SIGTERM")
2023-08-24 23:53:32,106 stacktrace       L0045 ERROR| RuntimeError: Test interrupted by SIGTERM
2023-08-24 23:53:32,106 stacktrace       L0046 ERROR| 
2023-08-24 23:53:32,106 test             L0206 DEBUG| DATA (filename=output.expected) => NOT FOUND (data sources: variant, test, file)
2023-08-24 23:53:32,106 test             L0206 DEBUG| DATA (filename=stdout.expected) => NOT FOUND (data sources: variant, test, file)
2023-08-24 23:53:32,106 test             L0206 DEBUG| DATA (filename=stderr.expected) => NOT FOUND (data sources: variant, test, file)
2023-08-24 23:53:32,107 test             L0933 ERROR| ERROR 1-./dfuse/caching_check.py:DfuseCachingCheck.test_dfuse_caching_check;run-container-dfuse-hosts-ior-client_processes-pool-server_config-engines-0-storage-0-1-51d0 -> TestSetupFail: Test interrupted by SIGTERM
```
Let's now remove this in order to land the fix to `src/tests/ftest/util/apricot/apricot/test.py`.
```suggestion
```